PraceDomowe/PracaDomowa5/Wasiak_Aleksandra/Praca domowa 5.ipynb

```diff
@@ -0,0 +1,231 @@
+{
```

Comment:
<p>To nie jest dobrze, maksymalny score postu został znaleziony poprawnie, ale należało go porównać z zaakceptowaną odpowiedzią dla danego postu (należało dołączyć tabelką Posts do Posts left_on='AcceptedAnswerId', right_on='Id'). </p><p> </p><p><code>0.5 punktu</code></p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-ExploratoryDataAnalysis/pull/138/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>